apps/docs/content/guides/platform/migrating-and-upgrading-projects.mdx

```diff
@@ -93,6 +93,48 @@ If you upgrade to a paid plan while your project is paused, any expired one-clic
   src="/docs/img/guides/platform/paused-paid-tier.png"
 />
 
+#### Restoring a Downloaded Backup Locally
+
+Downloaded backups are plaintext SQL files that can be loaded into a Postgres instance using [psql](https://www.postgresql.org/docs/current/app-psql.html). You can use our open source build tooling if you would like to restore a backup into a local environment so you can interact with it. Here are the steps:
+
+Install the [nix](https://nixos.org/):
+
+```sh
+curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install
+```
+
+Next, we need to start postgres locally.
+
+If you know the project's Postgres version from your project's [infrastructure project settings](https://supabase.com/dashboard/project/_/settings/infrastructure) page, you can use it to spin up an identical image locally. Using an identical image improves the odds of successfully restoring your project because all of the extension versions and configuration will match exactly.
```

Comment:
Is this infrastructure page available when the project is paused / not restorable? Do we embed this data elsewhere in the backup?

Reply:
No its not
I asked Joshen to add the image version next to the download button and he already knocked it out so I'll update it here

Reply:
done